src/Kafka/ManageKafkaPermissions/components/ResourcePrefix.tsx

```diff
@@ -0,0 +1,38 @@
+import { FormGroup, TextInput, ValidatedOptions } from "@patternfly/react-core";
+import { useTranslation } from "react-i18next";
+
+export type ResourcePrefixProps = {
+  value: string | undefined;
+  onChangeValue: (value: string | undefined) => void;
```

Comment:
```suggestion
  onChange: (value: string | undefined) => void;
```

Nitpicking. No need to repeat what's being changed, since it's the only change event triggered by this component (a component with more than one would be weird really)